scripts/install_tools.js

```diff
@@ -18,16 +18,16 @@ try {
         execute`./bootstrap.sh`;
         execute`./b2 -j8 --with-program_options --with-filesystem --with-system install `;
 
-        console.log("-- Installing Flatbuffers");
-        execute`mkdir -p /usr/local`;
-        process.chdir("/usr/local");
-        execute`git clone --depth 1 --branch v1.12.1 https://github.com/google/flatbuffers.git`;
-        process.chdir("/usr/local/flatbuffers");
-        execute`cmake -G "Unix Makefiles"`;
-        execute`make`;
-        execute`cp -r /usr/local/flatbuffers/include/flatbuffers /usr/local/include`;
-        execute`ln -s /usr/local/flatbuffers/flatc /usr/local/bin/flatc`;
-        execute`chmod +x /usr/local/flatbuffers/flatc`;
+        // console.log("-- Installing Flatbuffers");
+        // execute`mkdir -p /usr/local`;
+        // process.chdir("/usr/local");
+        // execute`git clone --depth 1 --branch v1.12.1 https://github.com/google/flatbuffers.git`;
+        // process.chdir("/usr/local/flatbuffers");
+        // execute`cmake -G "Unix Makefiles"`;
+        // execute`make`;
+        // execute`cp -r /usr/local/flatbuffers/include/flatbuffers /usr/local/include`;
+        // execute`ln -s /usr/local/flatbuffers/flatc /usr/local/bin/flatc`;
+        // execute`chmod +x /usr/local/flatbuffers/flatc`;
```

Comment:
Delete >>> Comment out

Reply:
https://github.com/finos/perspective/commit/1ec65f0388e1ce3f8b6f7e67f5142613a0cc5af0

I wasn't sure if we were keeping it around for some reason

Reply:
ive deleted this, and all the other commented flatbuffers references.